tests/phpunit/tests/TinyMCE.php

```diff
@@ -83,6 +83,16 @@ public function test_web_stories_tinymce_root_element() {
 		$this->assertSame( '<div id="web-stories-tinymce"></div>', $result );
 	}
 
+	/**
+	 * @covers ::script_loader_tag
+	 */
+	public function test_script_loader_tag() {
+		$tinymce = new \Google\Web_Stories\TinyMCE();
+		$result  = $tinymce->script_loader_tag( '<script></script>', 'another-handle', 'http://www.example.com' );
+
+		$this->assertSame( '<script></script>', $result );
```

Comment:
Uhm, can we render an empty string instead of an empty script tag?

Reply:
Fixed in https://github.com/google/web-stories-wp/pull/6781/commits/31eff36f8d6c2756b3571702b48a1646aaa69994